src/v2/view-builder/views/ov/ChallengeOktaVerifyPushView.js

```diff
@@ -36,6 +36,9 @@ const Body = BaseForm.extend(Object.assign(
           e.preventDefault();
         }
       }));
+      this.add(
```

Comment:
With OIE flows, this label is **always** disabled as the push is sent **automatically**. When we encounter an error state (i.e. the user rejects the notification) - this button is re-enabled with text matching the state.

Reply:
Just for my curiosity, isn't there a way to tell screen reader that there is disable button and reads loud the text?
Adding extra element for a11y seems hacky (I'm very new to a11y world. :D ) ?

Reply:
Good question! As a user navigates the page using a reader, a `disabled` button will never receive focus. As a result, all text within a `disabled` state (input, button, etc) will not be read aloud.

There are a few approaches to solve this problem:

1. Utilize a hidden element with an `alert` or `notify` role (what we're doing here). While it may seem "hacky" - this forces a screen reader to see updated text and read it aloud.
2. Style inputs/buttons accordingly without marking them as `disabled`. There is a great [article here](https://medium.com/@izzmo/anti-pattern-using-disabled-or-read-only-for-form-input-e2c24c669f5b) that goes a bit deeper into the accessibility concerns with marking elements as `disabled`. This option nests too many changes and creates more of an opportunity to release a regression.
3. Move visual messaging out of buttons (Odyssey approach). There are valid use cases where users should be informed of an action taking place after interacting with a page. For this scenario, a banner or toast could be leveraged to notify the user about the sent push notification. When we tackle a11y with our redesign/rewrite on Odyssey, we'll explore a more suitable UX.

Reply:
Do we need to remove it if the user rejects the push or if it times out?

Reply:
@mauriciocastillosilva-okta - We don't need to remove the new element, as rejecting the push will render the `ChallengeOktaVerifyResendPushView` view and the time-out flow does not enable the button. As a result, this a11y hidden element can remain present in the view as long as we keep this behavior. 😄